### PR TITLE
Need to make sure labels applied to /dev

### PR DESCRIPTION
**Pull request by rhatdan**
Signed-off-by: Dan Walsh <dwalsh@redhat.com>

### CODE REVIEW

File: libcontainer/rootfs_linux.go [hunk 1]
```diff
@@ -69,6 +69,9 @@ func setupRootfs(config *configs.Config, console *linuxConsole, pipe io.ReadWrit
 		if err := setupDevSymlinks(config.Rootfs); err != nil {
 			return newSystemErrorWithCause(err, "setting up /dev symlinks")
 		}
+		if err := label.Relabel(filepath.Join(config.Rootfs+"/dev"), config.MountLabel, false); err != nil {
```
Comment (cyphar) [inline libcontainer/rootfs_linux.go]: Did you mean to do `filepath.Join(config.Rootfs, "dev")`?

Comment (rhatdan) [inline libcontainer/rootfs_linux.go]: Does it matter?

Comment (rhatdan) [inline libcontainer/rootfs_linux.go]: Oops doing a little too much python

**Comment by cyphar** [comment]
LGTM.

**Comment by mrunalp** [comment]
LGTM

**Comment by rhatdan** [comment]
@mrunalp @crosbymichael This patch should be reverted.  /dev is mounted on a tmpfs with context="system_u:object_r:svirt_sandbox_file_t:s0:c1,c2".  Having this label will cause this relabel to fail with operation not supported.

I am not sure how to get the other error to happen with /dev not being on a tmpfs. 

Please revert this patch, or should I open another pull to remove it.

**Pull request merged**